tabpfn/scripts/transformer_prediction_interface.py

```diff
@@ -88,22 +90,27 @@ def check_file(e):
 
 
     #print(f'Loading {model_file}')
-
-    model, c = load_model(base_path, model_file, device, eval_positions=[], verbose=False)
+    if onlyInference:
+        print("Loading model that can be used for inference only")
+        model, c = load_model_only_inference(base_path, model_file, device)
+    else:
+        #until now also only capable of inference
+        model, c = load_model(base_path, model_file, device, eval_positions=[], verbose=False)
+    #model, c = load_model(base_path, model_file, device, eval_positions=[], verbose=False)
 
     return model, c, results_file
 
 
 class TabPFNClassifier(BaseEstimator, ClassifierMixin):
 
     def __init__(self, device='cpu', base_path=pathlib.Path(__file__).parent.parent.resolve(), model_string='', i=0, N_ensemble_configurations=3
-                 , combine_preprocessing=False, no_preprocess_mode=False, multiclass_decoder='permutation', feature_shift_decoder=True):
+                 , combine_preprocessing=False, no_preprocess_mode=False, multiclass_decoder='permutation', feature_shift_decoder=True, onlyInference=False):
```

Comment:
A minimal test (in the style of the `prior-fitting` test) that shows that this option does not change the predictions, not even the p's in the best case